pkg/bloomgateway/bloomgateway.go

```diff
@@ -299,6 +299,18 @@ func (g *Gateway) FilterChunkRefs(ctx context.Context, req *logproto.FilterChunk
 		return nil, err
 	}
 
+	// start time == end time => empty response
```

Comment:
nit: The `=>` here reads like greater-equal rather than an arrow. Can you use `-->` instead?